v3.4.19

rec opened the pull request:
When you approve the other pull request, I'll rebase that in, then run the full test suite with hardware and everything on that.

adammhaile commented:
Unless this gets a LOT more testing (because of the serial changes and all the rest of it) the next release will be 3.5, the docs release as far as I'm concerned. 
Including this much in a sub-minor release is a bit scary.

adammhaile commented:
Late?

rec commented:
It's a bit late to have put all these commits into separate releases in the
past.  :-D

On Mon, Jul 23, 2018 at 8:31 PM, Adam Haile <notifications@github.com>
wrote:

> Late?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-407156894>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPdsq_QQLtbksq1h6KDyUeC22QFchT9ks5uJhZsgaJpZM4VbdlG>
> .
>



-- 
     /t

*https://tom.ritchford.com <https://tom.ritchford.com>*
*https://tom.swirly.com <https://tom.swirly.com>*

adammhaile commented:
Well, yeah... but I'm not comfortable with releasing this. Maybe if I get some time to run some tests. But that's gonna be a few days.
I know you have the test platform now, which helps a lot. But I just want to do due diligence :P

rec commented:
Take yer time!  No one's clamoring for it!

On Mon, Jul 23, 2018 at 8:35 PM, Adam Haile <notifications@github.com>
wrote:

> Well, yeah... but I'm not comfortable with releasing this. Maybe if I get
> some time to run some tests. But that's gonna be a few days.
> I know you have the test platform now, which helps a lot. But I just want
> to do due diligence :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-407158222>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPdsll6TpexMschjBzVpVC-aX16NitLks5uJhd5gaJpZM4VbdlG>
> .
>



-- 
     /t

*https://tom.ritchford.com <https://tom.ritchford.com>*
*https://tom.swirly.com <https://tom.swirly.com>*

adammhaile commented:
It's all good. I've had zero time to test either. Hopefully soon....

On Fri, Aug 3, 2018 at 5:18 AM Tom Swirly <notifications@github.com> wrote:

> I haven't pushed on this because I haven't had time to look at #892
> <https://github.com/ManiacalLabs/BiblioPixel/issues/892>.
>
> However, now we have Python 3.7 support, I think we should release (when
> you get a chance to test on your end) and not worry about #892
> <https://github.com/ManiacalLabs/BiblioPixel/issues/892>, which only
> affects animations contained in Parallel animations like the Mixer (and
> has been there since before the last release anyway).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-410196481>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6a6voBR2OLaPTkjZy0s7FyuSRdJW9sks5uNBVKgaJpZM4VbdlG>
> .
>

rec commented:
All right.  I just rebased and updated the CHANGELIST.md.

Let's plan a time when we can run the tests again - give me a bit of advance warning so I can be completely prepped - and get this out!

adammhaile commented:
I can work on it tomorrow or saturday for a bit.

On Thu, Aug 30, 2018 at 4:45 AM Tom Swirly <notifications@github.com> wrote:

> All right. I just rebased and updated the CHANGELIST.md.
>
> Let's plan a time when we can run the tests again - give me a bit of
> advance warning so I can be completely prepped - and get this out!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-417240036>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA6a6rT0K0fMcUpZi1ntgRwH_Def_h2sks5uV6YtgaJpZM4VbdlG>
> .
>

rec commented:
Both days are good, tomorrow is excellent.

I'll make sure to have a good, tested version up for you by tomorrow
morning!

On Thu, Aug 30, 2018 at 12:56 PM Adam Haile <notifications@github.com>
wrote:

> I can work on it tomorrow or saturday for a bit.
>
> On Thu, Aug 30, 2018 at 4:45 AM Tom Swirly <notifications@github.com>
> wrote:
>
> > All right. I just rebased and updated the CHANGELIST.md.
> >
> > Let's plan a time when we can run the tests again - give me a bit of
> > advance warning so I can be completely prepped - and get this out!
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-417240036
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AA6a6rT0K0fMcUpZi1ntgRwH_Def_h2sks5uV6YtgaJpZM4VbdlG
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManiacalLabs/BiblioPixel/pull/891#issuecomment-417278179>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPdsvYSFRAgCIWrjTnVaBO-ocsOh2jNks5uV8TggaJpZM4VbdlG>
> .
>


-- 
     /t

*https://tom.ritchford.com <https://tom.ritchford.com>*
*https://tom.swirly.com <https://tom.swirly.com>*
